### PR TITLE
Invest flow tweaks for low liqudity protection

### PR DESCRIPTION
**Pull request by evgenyboxer**
# Description

Adds a couple of warnings/errors for low liquidity pools - according to "quick wins" design here: https://www.figma.com/file/S5p30cxomV6gIyRZJMrvb3/Add-liquidity?node-id=802%3A6347

1. Added a "low liquidity" warning if the total liquidity is less than 50K.
2. Added "zero amount" assets to be shown in the preview screen (along with sorting them by their fiat value).
3. Added a new "rekt" error that disallows the user to continue.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Easiest to test is to tweak the values locally to see the different warnings/errors.

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have requested at least 2 reviews (If the PR is significant enough, use best judgement here)
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] If package-lock.json has changes, it was intentional.
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/components/forms/pool_actions/InvestForm/InvestForm.vue [hunk 1]
```diff
@@ -210,13 +216,21 @@ watch(useNativeAsset, shouldUseNativeAsset => {
     <BalAlert
       v-if="forceProportionalInputs"
       type="warning"
-      :title="$t('investment.warning.managedPoolTradingHaulted.title')"
+      :title="$t('investment.warning.managedPoolTradingHalted.title')"
```
Comment (evgenyboxer) [inline src/components/forms/pool_actions/InvestForm/InvestForm.vue:219]: Fixed a typo

File: src/components/forms/pool_actions/InvestForm/components/InvestPreviewModal/InvestPreviewModal.vue [hunk 2]
```diff
@@ -67,7 +69,7 @@ const amountMap = computed(
   (): AmountMap => {
     const amountMap = {};
     fullAmounts.value.forEach((amount, i) => {
-      if (hasAmount(i)) amountMap[props.tokenAddresses[i]] = amount;
+      amountMap[props.tokenAddresses[i]] = amount;
```
Comment (evgenyboxer) [inline src/components/forms/pool_actions/InvestForm/components/InvestPreviewModal/InvestPreviewModal.vue:72]: There is no amount check anymore

File: src/constants/poolLiquidity.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const LOW_LIQUIDITY_THRESHOLD = 50_000;
+export const HIGH_PRICE_IMPACT = 0.01;
+export const REKT_PRICE_IMPACT = 0.5;
```
Comment (evgenyboxer) [inline src/constants/poolLiquidity.ts:3]: I think we can start with 50% as rekt price impact.

File: src/constants/poolAPR.ts [hunk 4]
```diff
@@ -1,3 +1,3 @@
 // Do not display APR values greater than this amount; they are likely to be nonsensical
 // These can arise from pools with extremely low balances (e.g., completed LBPs)
-export const APR_THRESHOLD = 10000;
+export const APR_THRESHOLD = 10_000;
```
Comment (evgenyboxer) [inline src/constants/poolAPR.ts:3]: more readable

**Review by 1saf — APPROVED** [review]

**Review by timjrobinson — APPROVED** [review]

File: src/components/forms/pool_actions/InvestForm/components/InvestPreviewModal/InvestPreviewModal.vue (hunk 2 above)
Comment (evgenyboxer) [reply src/components/forms/pool_actions/InvestForm/components/InvestPreviewModal/InvestPreviewModal.vue:72]: Pon wanted to show tokens - even when the amount is `0`

Comment (garethfuller) [reply src/components/forms/pool_actions/InvestForm/components/InvestPreviewModal/InvestPreviewModal.vue:72]: Ahh right, this is the preview modal, got it.

**Comment by evgenyboxer** [comment]
@garethfuller I've changed it

**Review by garethfuller — APPROVED** [review]

**Pull request merged**